[2016-12-19 11:20:25,803] INFO [Group Metadata Manager on Broker 0]: Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.GroupMetadataManager)
[2016-12-19 11:30:25,803] INFO [Group Metadata Manager on Broker 0]: Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.GroupMetadataManager)
[2016-12-19 11:35:23,310] INFO [Kafka Server 0], shutting down (kafka.server.KafkaServer)
[2016-12-19 11:35:23,349] INFO [Kafka Server 0], Starting controlled shutdown (kafka.server.KafkaServer)
[2016-12-19 11:35:54,976] ERROR Uncaught exception in scheduled task 'kafka-log-retention' (kafka.utils.KafkaScheduler)
java.io.FileNotFoundException: /tmp/kafka-logs/winagg-0/00000000000000000000.index (Datei oder Verzeichnis nicht gefunden)
	at java.io.RandomAccessFile.open0(Native Method)
	at java.io.RandomAccessFile.open(RandomAccessFile.java:316)
	at java.io.RandomAccessFile.<init>(RandomAccessFile.java:243)
	at kafka.log.OffsetIndex$$anonfun$resize$1.apply(OffsetIndex.scala:277)
	at kafka.log.OffsetIndex$$anonfun$resize$1.apply(OffsetIndex.scala:276)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:262)
	at kafka.log.OffsetIndex.resize(OffsetIndex.scala:276)
	at kafka.log.OffsetIndex$$anonfun$trimToValidSize$1.apply$mcV$sp(OffsetIndex.scala:265)
	at kafka.log.OffsetIndex$$anonfun$trimToValidSize$1.apply(OffsetIndex.scala:265)
	at kafka.log.OffsetIndex$$anonfun$trimToValidSize$1.apply(OffsetIndex.scala:265)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:262)
	at kafka.log.OffsetIndex.trimToValidSize(OffsetIndex.scala:264)
	at kafka.log.Log.roll(Log.scala:627)
	at kafka.log.Log.deleteOldSegments(Log.scala:549)
	at kafka.log.LogManager.kafka$log$LogManager$$cleanupExpiredSegments(LogManager.scala:421)
	at kafka.log.LogManager$$anonfun$cleanupLogs$3.apply(LogManager.scala:452)
	at kafka.log.LogManager$$anonfun$cleanupLogs$3.apply(LogManager.scala:450)
	at scala.collection.TraversableLike$WithFilter$$anonfun$foreach$1.apply(TraversableLike.scala:772)
	at scala.collection.Iterator$class.foreach(Iterator.scala:727)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1157)
	at scala.collection.IterableLike$class.foreach(IterableLike.scala:72)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
	at scala.collection.TraversableLike$WithFilter.foreach(TraversableLike.scala:771)
	at kafka.log.LogManager.cleanupLogs(LogManager.scala:450)
	at kafka.log.LogManager$$anonfun$startup$1.apply$mcV$sp(LogManager.scala:190)
	at kafka.utils.KafkaScheduler$$anonfun$1.apply$mcV$sp(KafkaScheduler.scala:110)
	at kafka.utils.CoreUtils$$anon$1.run(CoreUtils.scala:60)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[2016-12-19 11:38:11,776] INFO [Kafka Server 0], Controlled shutdown succeeded (kafka.server.KafkaServer)
[2016-12-19 11:38:11,783] INFO [Socket Server on Broker 0], Shutting down (kafka.network.SocketServer)
[2016-12-19 11:38:11,797] INFO [Socket Server on Broker 0], Shutdown completed (kafka.network.SocketServer)
[2016-12-19 11:38:11,798] INFO [Kafka Request Handler on Broker 0], shutting down (kafka.server.KafkaRequestHandlerPool)
[2016-12-19 11:38:11,805] INFO [Kafka Request Handler on Broker 0], shut down completely (kafka.server.KafkaRequestHandlerPool)
[2016-12-19 11:38:11,809] INFO [ThrottledRequestReaper-Produce], Shutting down (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2016-12-19 11:38:12,139] INFO [ThrottledRequestReaper-Produce], Stopped  (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2016-12-19 11:38:12,141] INFO [ThrottledRequestReaper-Produce], Shutdown completed (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2016-12-19 11:38:12,141] INFO [ThrottledRequestReaper-Fetch], Shutting down (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2016-12-19 11:38:12,227] INFO [ThrottledRequestReaper-Fetch], Stopped  (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2016-12-19 11:38:12,227] INFO [ThrottledRequestReaper-Fetch], Shutdown completed (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2016-12-19 11:38:12,227] INFO [KafkaApi-0] Shutdown complete. (kafka.server.KafkaApis)
[2016-12-19 11:38:12,228] INFO [Replica Manager on Broker 0]: Shutting down (kafka.server.ReplicaManager)
[2016-12-19 11:38:12,231] INFO [ReplicaFetcherManager on broker 0] shutting down (kafka.server.ReplicaFetcherManager)
[2016-12-19 11:38:12,232] INFO [ReplicaFetcherManager on broker 0] shutdown completed (kafka.server.ReplicaFetcherManager)
[2016-12-19 11:38:12,232] INFO [ExpirationReaper-0], Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2016-12-19 11:38:12,270] INFO [ExpirationReaper-0], Stopped  (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2016-12-19 11:38:12,270] INFO [ExpirationReaper-0], Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2016-12-19 11:38:12,270] INFO [ExpirationReaper-0], Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2016-12-19 11:38:12,473] INFO [ExpirationReaper-0], Stopped  (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2016-12-19 11:38:12,473] INFO [ExpirationReaper-0], Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2016-12-19 11:38:12,489] INFO [Replica Manager on Broker 0]: Shut down completely (kafka.server.ReplicaManager)
[2016-12-19 11:38:12,490] INFO Shutting down. (kafka.log.LogManager)
[2016-12-19 11:38:12,529] WARN /tmp/kafka-logs/winagg-0/00000000000000000000.index (Datei oder Verzeichnis nicht gefunden) (kafka.utils.CoreUtils$)
java.io.FileNotFoundException: /tmp/kafka-logs/winagg-0/00000000000000000000.index (Datei oder Verzeichnis nicht gefunden)
	at java.io.RandomAccessFile.open0(Native Method)
	at java.io.RandomAccessFile.open(RandomAccessFile.java:316)
	at java.io.RandomAccessFile.<init>(RandomAccessFile.java:243)
	at kafka.log.OffsetIndex$$anonfun$resize$1.apply(OffsetIndex.scala:277)
	at kafka.log.OffsetIndex$$anonfun$resize$1.apply(OffsetIndex.scala:276)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:262)
	at kafka.log.OffsetIndex.resize(OffsetIndex.scala:276)
	at kafka.log.OffsetIndex$$anonfun$trimToValidSize$1.apply$mcV$sp(OffsetIndex.scala:265)
	at kafka.log.OffsetIndex$$anonfun$trimToValidSize$1.apply(OffsetIndex.scala:265)
	at kafka.log.OffsetIndex$$anonfun$trimToValidSize$1.apply(OffsetIndex.scala:265)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:262)
	at kafka.log.OffsetIndex.trimToValidSize(OffsetIndex.scala:264)
	at kafka.log.OffsetIndex.close(OffsetIndex.scala:336)
	at kafka.log.LogSegment$$anonfun$close$1.apply$mcV$sp(LogSegment.scala:270)
	at kafka.utils.CoreUtils$.swallow(CoreUtils.scala:79)
	at kafka.utils.Logging$class.swallowWarn(Logging.scala:92)
	at kafka.utils.CoreUtils$.swallowWarn(CoreUtils.scala:51)
	at kafka.utils.Logging$class.swallow(Logging.scala:94)
	at kafka.utils.CoreUtils$.swallow(CoreUtils.scala:51)
	at kafka.log.LogSegment.close(LogSegment.scala:270)
	at kafka.log.Log$$anonfun$close$2.apply(Log.scala:291)
	at kafka.log.Log$$anonfun$close$2.apply(Log.scala:290)
	at scala.collection.Iterator$class.foreach(Iterator.scala:727)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1157)
	at scala.collection.IterableLike$class.foreach(IterableLike.scala:72)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
	at kafka.log.Log.close(Log.scala:290)
	at kafka.log.LogManager$$anonfun$shutdown$3$$anonfun$6$$anonfun$apply$2.apply$mcV$sp(LogManager.scala:237)
	at kafka.utils.CoreUtils$$anon$1.run(CoreUtils.scala:60)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[2016-12-19 11:38:12,533] INFO Shutdown complete. (kafka.log.LogManager)
[2016-12-19 11:38:12,533] INFO [GroupCoordinator 0]: Shutting down. (kafka.coordinator.GroupCoordinator)
[2016-12-19 11:38:12,533] INFO [ExpirationReaper-0], Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2016-12-19 11:38:12,672] INFO [ExpirationReaper-0], Stopped  (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2016-12-19 11:38:12,672] INFO [ExpirationReaper-0], Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2016-12-19 11:38:12,672] INFO [ExpirationReaper-0], Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2016-12-19 11:38:12,809] INFO [ExpirationReaper-0], Stopped  (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2016-12-19 11:38:12,809] INFO [ExpirationReaper-0], Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2016-12-19 11:38:12,810] INFO [GroupCoordinator 0]: Shutdown complete. (kafka.coordinator.GroupCoordinator)
[2016-12-19 11:38:12,830] INFO [Kafka Server 0], shut down completed (kafka.server.KafkaServer)
